libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -1519,8 +1519,11 @@ MAV_RESULT AP_InertialSensor::calibrate_trim()
         goto failed;
     }
 
+#if AP_AHRS_ENABLED
```

Comment:
this is now inside `#if HAL_GCS_ENABLED && AP_AHRS_ENABLED` ? SO we should not need to check again?

Reply:
Fixed